### PR TITLE
Adding optional seeding

### PR DESCRIPTION
**Pull request by KrainskiL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ It contains the following files:
 The main file intended to be used is `abcd_sampler.jl`.
 Here is an example configuration file, named `example_config.toml`, in this guide:
 ```
+seed = "-1"                   # RNG seed, use -1 for no seeding
```
Comment (bkamins) [inline README.md]:
```suggestion
seed = ""                     # RNG seed, use "" for no seeding
```

File: utils/example_config.toml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+seed = "-1"                   # RNG seed, use -1 for no seeding
```
Comment (bkamins) [inline utils/example_config.toml]:
```suggestion
seed = ""                     # RNG seed, use "" for no seeding
```

File: utils/abcd_sampler.jl [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 using ABCDGraphGenerator
+using Random
 
 @info "Usage: julia abcd_sampler.jl config_filename"
 @info "For the syntax of config_filename see example_config.toml file"
 
 filename = ARGS[1]
 conf = Pkg.TOML.parsefile(filename)
+seed = parse(Int, conf["seed"])
+seed != -1 && Random.seed!(seed)
```
Comment (bkamins) [inline utils/abcd_sampler.jl]:
```suggestion
isempty(conf["seed"]) || Random.seed!(parse(Int, conf["seed"]))
```

**Comment by bkamins** [comment]
Thank you. Can you please check if my fix is OK? If yes - please commit the suggestions and I will merge it.

**Comment by KrainskiL** [comment]
Fix is working as expected.

**Comment by bkamins** [comment]
Maybe also add seeding to the generators of degrees etc. as it seems Arash also used them?

**Pull request merged**